docs/commands.md

```diff
@@ -48,8 +48,58 @@ TS.CREATE temperature:2:32 RETENTION 60000 DUPLICATE_POLICY MAX LABELS sensor_id
 
 ### DEL
 
-A series can be deleted using redis `DEL` command. Timeout can be set for a series using
-redis `EXPIRE` command.
+A series can be deleted using redis [`DEL`](https://redis.io/commands/del) command.
+
+Timeout can be set for a series using redis [`EXPIRE`](https://redis.io/commands/expire) command.
+
+
+```sql
+DEL key
+```
+
+* key - Key name for timeseries
+
+#### Complexity
+
+DEL complexity is O(N) where N is the number of keys that will be removed.
```

Comment:
@filipecosta90 if this is the case, we need to define the command as so you can pass several keys to it